session1/challenge/client/src/js/components/container/App.js

```diff
@@ -27,28 +31,34 @@ const DataTile = ({ name, value, loading, active, onClick }) => {
 
 const DataList = ({ data, loading, onClick, principal}) => {
   const items = loading ? FAKE_DATA : data;
-  // TODO: Use the DateTile and return an array for every data
-
+  
   /* value={items[key]}
    * name={key}
    * onClick={onClick}
    * loading={loading}
    * active={key === principal}*/
-
-  return null
+  return Object.entries(items).map(([key, value]) =>{
+    return (
+      <TileErrorBoundary key={key}>
+        <DataTile 
+          value={value}
+          name={key}
+          onClick={onClick}
+          loading={loading}
+          active={key === principal} />
+      </TileErrorBoundary>
+    );
+  });
 }
 
 const InformationModal = ({ open, onClick, children }) => {
-  // TODO: Create a Portal component and wrap a modal
-  // Create the portal on the ../presentational/ModalPortal.js file
-
   const modal = (
       <Modal open={open} onClick={onClick}>
         {children}
       </Modal>
   );
 
-  return null;
+  return ReactDOM.createPortal(modal, document.getElementsByTagName('body')[0]);;
```

Comment:
There is a file named `ModalPortal.js` inside the `presentational` folder. Try to implement the `Portal` component there.

Reply:
Ok, working on that